tests/vcs/git/GitCommitBearTest.py

```diff
@@ -16,6 +16,79 @@
 from coalib.settings.Setting import Setting
 
 
+@generate_skip_decorator(GitCommitBear)
+class FakeGitHubMergeCommitBear(unittest.TestCase):
+
+    def run_uut(self, *args, **kwargs):
+        """
+        Runs the unit-under-test (via `self.uut.run()`) and collects the
+        messages of the yielded results as a list.
+
+        :param args:   Positional arguments to forward to the run function.
+        :param kwargs: Keyword arguments to forward to the run function.
+        :return:       A list of the message strings.
+        """
+        return list(result.message for result in self.uut.run(*args, **kwargs))
+
+    def setUp(self):
+        self.msg_queue = Queue()
+        self.section = Section('')
+        self.uut = GitCommitBear(None, self.section, self.msg_queue)
+
+        self._old_cwd = os.getcwd()
+        self.gitdir = mkdtemp()
+        os.chdir(self.gitdir)
+        GitCommitBearTest.run_git_command('init')
+        GitCommitBearTest.run_git_command(
+            'config', 'user.email coala@coala.io')
+        GitCommitBearTest.run_git_command('config', 'user.name coala')
+
+    def test_github_PR_merge_commit_offline(self):
+        run_shell_command('touch "a" >> testfile1.txt')
+        run_shell_command('git add testfile1.txt')
+        run_shell_command('git commit -m "First commit"')
+        commit_hash1, _ = run_shell_command('git rev-parse HEAD')
+        commit_hash1 = commit_hash1.strip('\n')
+
+        run_shell_command('git checkout -b feature1 master')
+        run_shell_command('touch "b" >> testfile2.txt')
+        run_shell_command('git add testfile2.txt')
+        run_shell_command('git commit -m "Second commit"')
+        commit_hash2, _ = run_shell_command('git rev-parse HEAD')
+        commit_hash2 = commit_hash2.strip('\n')
+
+        run_shell_command('git checkout master')
+        run_shell_command('git merge --no-ff feature1')
+        command = 'git commit --amend -m ' + '\"Merge ' + \
```

Comment:
Don't use `\` please, use parentheses instead, `"` doesn't need to be escaped if we're using `'` string style..
I think it would be good to use string interpolation here..